setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = acstools
-version = 2.0.1
+version = 2.0.2
```

Comment:
This should also be updated in `defsetup.py` for consistency although I don't think that file is used.